src/community/community.mongo.service.ts

```diff
@@ -51,6 +51,27 @@ export class CommunityMongoService {
         }
     }
 
+    // TODO: find better solution for this
+    async fetchCommunityByDiscordGuildId(guildId: string): Promise<Community> {
```

Comment:
should update return type to include null/empty

Reply:
This actually throws an http exception if it doesn't find anything. And the client automatically adds undefined to all types